apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/MetadataInfoTabs/index.tsx

```diff
@@ -28,7 +30,7 @@ export const MetadataInfoTabs = forwardRef<HTMLDivElement, Props>(
         ref={ref}
         className={cn(
           'flex flex-col flex-grow min-h-0 bg-background',
-          'items-center relative',
+          'border border-border rounded-lg items-center relative',
```

Comment:
Try not to remove behaviour/style from a component used in different places. 
Removing border from it introduced this visual regression on llm evaluations table
![image](https://github.com/user-attachments/assets/c4e40df6-cb62-40ef-9642-c79a8f6019e9)

For me is better to increase the surface of a shared component. Like in this case adding `bottomActions` to hold that new UI piece inside this existing component

![image](https://github.com/user-attachments/assets/8cc22b5e-38d9-49b7-b1d2-fc2fd1247286)

Reply:
🐛 ☝️ Visual bug fixed